tsdb/engine/tsm1/cache.go

```diff
@@ -561,7 +561,10 @@ func (c *Cache) values(key string) Values {
 	if e == nil {
 		return nil
 	}
-	return e.values
+	e.mu.RLock()
```

Comment:
The method comment should probably be updated to reflect the fact a lock is now taken.